### PR TITLE
Add _translation_trans_equations method.

### PR DESCRIPTION
**Pull request by szymag**
This PR introduces transformation equations for translations.

### CODE REVIEW

**Comment by szymag** [comment]
ping @Upabjojr 
Do you like parameter `inverse`? I think that it's better than separate method.

File: sympy/vector/tests/test_coordsysrect.py [hunk 1]
```diff
@@ -399,3 +399,17 @@ def test_rotation_trans_equations():
...
+def test_translation_trans_equations():
+    from sympy import symbols
+    q0 = symbols('q0')
+    a = CoordSys3D('a')
+    assert a._translation_trans_equations() == (a.x, a.y, a.z)
+    b = a.orient_new_axis('b', 0, -a.k)
+    assert b._translation_trans_equations() == (b.x, b.y, b.z)
+    assert b._translation_trans_equations(inverse=True) == (b.x, b.y, b.z)
+    c = a.orient_new_axis('c', q0, -a.k, location=(a.i + a.j + a.k))
+    assert c._translation_trans_equations() == (c.x + 1, c.y + 1, c.z + 1)
```
Comment (Upabjojr) [inline sympy/vector/tests/test_coordsysrect.py]: Well, does this transformation equation alone make sense? You are rotating and translating the coordinate system, I would expect a combined transformation equation. I would suggest to translate only for this test (use `.locate_new`).

**Comment by Upabjojr** [comment]
> Do you like parameter inverse?

OK

File: sympy/vector/tests/test_coordsysrect.py (hunk 1 above)
Comment (szymag) [inline sympy/vector/tests/test_coordsysrect.py]: OK.
When this PR and #13021 will be merged I will add some other tests for composition including translation.

Comment (Upabjojr) [inline sympy/vector/tests/test_coordsysrect.py]: OK, we need something more generic anyway.

Comment (szymag) [inline sympy/vector/tests/test_coordsysrect.py]: Do you mean more generic than #13021?

Comment (Upabjojr) [inline sympy/vector/tests/test_coordsysrect.py]: We need to find a good API now. I suggest to change the `CoordSys3D` constructor.

**Pull request merged**